data/transactions/logic/README.md

```diff
@@ -403,9 +403,9 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | `args` | Ath LogicSig argument |
 | `txn f` | field F of current transaction |
 | `gtxn t f` | field F of the Tth transaction in the current group |
-| `txna f i` | Ith value of the array field F of the current transaction |
+| `txna f i` | Ith value of the array field F of the current transaction<br />txna can be called using txn with 2 immediates. |
```

Comment:
I'm not entirely sure how to test this and also if you're talking about TEAL.pdf, it looks like it can't handle most tables in the first place (at least for the newest release on the specs repo)

Reply:
It was mucked up a while back and I fixed it somewhat. Perhaps it's messed up again.  It's nothing for you to worry about, I think. But one of my other comments notes that the html ends up elsewhere that it doesn't belong.